addons/website_crm_partner_assign/wizard/crm_forward_to_partner.py

```diff
@@ -113,7 +113,7 @@ def get_lead_portal_url(self, lead):
             action_ref and action_ref.id or False)
         return portal_link
 
-    def get_portal_url(self):
+    def _get_base_portal_url(self):
         if self:
             portal_url = self[0].get_base_url()
         else:
```

Comment:
Is it possible to have no record when rendering the mail template where this is called?
Can't you drop this method and use `get_base_url` in the mail template :thinking: ?

Reply:
This is wild. And seems legit :D .